packages/web/src/pages/settings-page/components/desktop/ManagerMode/AccountListItem/ManagerListItem.tsx

```diff
@@ -0,0 +1,174 @@
+import { useCallback, useMemo } from 'react'
+
+import { useAppContext } from '@audius/common/context'
+import { useIsManagedAccount } from '@audius/common/hooks'
+import { Name, UserManagerMetadata } from '@audius/common/models'
+import { useRemoveManager } from '@audius/common/src/api/account'
+import { accountSelectors, chatSelectors } from '@audius/common/store'
+import {
+  Flex,
+  IconButton,
+  IconKebabHorizontal,
+  IconMessage,
+  IconTrash,
+  IconUser,
+  PopupMenu,
+  Text
+} from '@audius/harmony'
+
+import { useNavigateToPage } from 'hooks/useNavigateToPage'
+import { useComposeChat } from 'pages/chat-page/components/useComposeChat'
+import { useSelector } from 'utils/reducer'
+import { profilePage } from 'utils/route'
+import zIndex from 'utils/zIndex'
+
+import { ArtistInfo } from './ArtistInfo'
+
+const { getUserId } = accountSelectors
+const { getCanCreateChat } = chatSelectors
+
+const messages = {
+  moreOptions: 'more options',
+  removeManager: 'Remove Manager',
+  visitProfile: 'Visit Profile',
+  sendMessage: 'Send Message',
+  invitePending: 'Invite Pending',
+  cancelInvite: 'Cancel Invite'
+}
+
+type ManagerListItemProps = {
+  data: UserManagerMetadata
+  onRemoveManager: (params: { userId: number; managerUserId: number }) => void
+}
+
+export const ManagerListItem = ({
+  data: { manager, grant },
```

Comment:
nit, but I think a more specific prop name would be preferable

Reply:
heard and agreed, will update!